app/assets/stylesheets/sul_variables.scss

```diff
@@ -46,4 +46,5 @@ $sul-link-hover-color: #00548f;
 $sul-metadata-bg: $color-beige-05;
 $sul-metadata-heading-color: $gray-54-percent;
 $sul-metadata-label-color: dimgrey;
-$sul-restricted-text-color: $color-cardinal-red;
\ No newline at end of file
+$sul-restricted-text-color: $color-cardinal-red;
+$sul-fog-light: #f4f4f4;
```

Comment:
I think this should be `stanford-fog-light` as the palette is defined by the university, not the library.

Reply:
Can it go into palette.scss?

Reply:
Yep. I renamed it color-fog-light in pallette because that conforms with what is in there. I think if we want to rename everything stanford we can in a separate PR but I would like to keep things in line for this one.